templates/bareos-dir/client/client.conf.j2

```diff
@@ -1,7 +0,0 @@
-# {{ ansible_managed }}
-Client {
-  Name = {{ item.name }}
-  Address = "{{ item.address }}"
-  Password = "{{ bareos_client_password }}"
-  Maximum Concurrent Jobs = 50
```

Comment:
So there is no loss of parameter by dropping this template? Will the `Maximum Concurrent Jobs` still be configured?

Reply:
No, the Maximum Concurrent Jobs won't be configured anymore at the client level (as originally done by this role). But it's still configured in the [job](https://github.com/mila-iqia/ansible-role-bareos/blob/mila/templates/bareos-dir/job/job.conf.j2#L6), [storage](https://github.com/mila-iqia/ansible-role-bareos/blob/mila/templates/bareos-dir/storage/storage.conf.j2#L8), [device](https://github.com/mila-iqia/ansible-role-bareos/blob/mila/templates/bareos-sd/device/device.conf.j2#L11). It would never have reached the client limit anyway (at least not while it's fixed at 50 everywhere)